wom/src/test/scala/wom/expression/PlaceholderWomExpression.scala

```diff
@@ -20,6 +20,8 @@ final case class PlaceholderWomExpression(inputs: Set[String], fixedWomType: Wom
 }
 
 case object PlaceholderIoFunctionSet extends IoFunctionSet {
+  val DefaultFileSize = -12345D
```

Comment:
this may be used at some point, had to make a decision concerning related code and waiting to see if it broke any tests before removing this artifact.